setup.py

```diff
@@ -62,14 +60,12 @@
     license='MIT license',
     zip_safe=False,
     keywords='azure iot edge dev tool',
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <3.8',
+    python_requires='!=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <3.8',
```

Comment:
>!=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.* [](start = 21, length = 52)

Will this be equivalent - '>=3.6'?

Reply:
yes indeed! you are right. 
It should be `python_requires = '>=3.6, <3.8'`